project/Settings.scala

```diff
@@ -188,6 +188,7 @@ object Settings {
       removeIntermediateContainers = BuildOptions.Remove.Always
     )
   )
+  val womSettings = List(addCompilerPlugin("org.spire-math" %% "kind-projector" % kindProjectorV))
```

Comment:
If we truly don't want other wom-ish languages to have ecmascript support<sup>1</sup>, and are moving the evaluator to `cwl.internal`, then the build should also be updated to match such that only the `cwl` and not `wom` sub-project uses `rhino`, the extra `kind-projector`, etc.

---

<sup>1</sup> ToL: While I love WDL's overall syntax, I'm curious if one day I'd prefer blocks of JS for WDL-functions instead of what we [have now](https://github.com/broadinstitute/cromwell/blob/a40de672c565c4bbd40f57ff96d4ee520dc2b4fc/centaur/src/main/resources/integrationTestCases/Somatic/Mutect2/Mutect2.wdl#L292). Still, ETA on this would be a long way out.